### PR TITLE
Settings in emu command and more

### PR DESCRIPTION
**Pull request by Lemmmy**
- Allow changing settings from the `emu` command (currently resolution and cursor char)
- Allow changing the resolution via the `ccemux` api
- Show the computer label in the window title

Please ignore the accidental reformat.

### CODE REVIEW

File: src/main/resources/programs/emu.lua [hunk 1]
```diff
@@ -39,6 +39,9 @@ if ccemux then
                         elseif args[3] == "pocket" then
                             ccemux.setResolution(26, 20)
                             print("Set resolution to pocket (26x20)")
+                        elseif args[3] == "turtle" then
```
Comment (Lemmmy) [inline src/main/resources/programs/emu.lua]: ✅

**Comment by Lignumm** [comment]
Ok :+1: :ok: :ok_hand:

**Comment by Lemmmy** [comment]
ok

**Comment by apemanzilla** [comment]
We still have the issue with config comments being lost when saving settings

**Comment by Lemmmy** [comment]
yes this branch isnt done yet just the pr

**Pull request merged**